src/query/v2/requests.hpp

```diff
@@ -57,24 +57,25 @@ struct EdgeId {
   Gid gid;
 };
 
-struct Vertex {
-  VertexId id;
-  std::vector<Label> labels;
-  friend bool operator==(const Vertex &lhs, const Vertex &rhs) {
-    return (lhs.id == rhs.id) && (lhs.labels == rhs.labels);
-  }
-};
-
 struct Edge {
   VertexId src;
   VertexId dst;
+  std::optional<std::vector<std::pair<PropertyId, Value>>> properties;
```

Comment:
Currently, we are inconsistent with this one.
At some places we are using `std::map<...>` and at others `std::vector<std::pair<...>>`. When the `Shard` class was reworked it took some parameters as `std::vector<std::pair<...>>` and so this was just how some fields were defined in the messages. This is not great of course, I have already talked with @kostasrim about that. Once the exact communication type is in place (or at least decided), we plan on reworking the messages.
Currently I would rather not spend time with that since the modification here would affect the `ShardRequestManager` as well.
If you insist of course I can change that in this PR.